packages/pwa-buildpack/src/WebpackTools/plugins/makeMagentoRootComponentsPlugin/index.js

```diff
@@ -110,16 +110,16 @@ class MagentoRootComponentsPlugin {
         );
 
         this.contents = `
-const rootComponentsMap = {
-${Object.entries(rootComponentImporters)
-    .map(entry => entry.join(':'))
-    .join(',\n')}
-};
-const key = ${toRootComponentMapKey.toString()};
-export default function fetchRootComponent(type, variant = 'default') {
-    return rootComponentsMap[key(type, variant)]().then(m => m.default || m);
-};
-`;
+            const rootComponentsMap = {
```

Comment:
Template strings will contain tabs/spaces. That's probably why this was not indented. We may want to undo this and add a `eslint ignore` or something along those lines here.